R/mcp-enable.R

```diff
@@ -0,0 +1,71 @@
+#' Start an MCP server for your R session
+#' @export
+mcp_enable <- function(host = "127.0.0.1", port = 8081) {
+  tryCatch({
+    existing_server <- httpuv::listServers()
+    if (length(existing_server) > 0) {
+      httpuv::stopServer(existing_server[[1]])
+    }
```

Comment:
Note that if the user started a Shiny or Plumber app, this code would stop that app.

I suggest doing something like this: create an environment called `globals` at the top level of the package with a member called `mcpserver` that starts out `NULL`, and then when you create the server, store it there:

```
  globals$mcpserver <- httpuv::startServer(...)
```

Then just check for that